docs/deployment.md

```diff
@@ -0,0 +1,91 @@
+
+This document details the process we use to deploy Page Shot to our stage and production environments.
+
+## Overview of schedule ##
+
+(you can read more below)
+
+- Thursday at 12pm PST the train is cut, all code is in. Style tweaks can be made after this point.
+- Friday at 9am PST we tag and start pushing to stage, then send email out.
+- Monday at regular stand-up we review anything that Softvision found and fix it.
+- Wednesday at ~8am PST we push and Softvision verifies.
+
+## Softvision ##
+
+Softvision is our embedded QA team. Their main functions are to write test plans and verify deployments.
+
+## Team Notification ##
+
+During the checkin before the end of the [current milestone](https://github.com/mozilla-services/pageshot/milestones), we will inform the team that we will be building a release against `master`.
+
+Note: we auto deploy the master branch to our *development environment*: [http://testpilot.dev.mozaws.net](https://pageshot.dev.mozaws.net)
+
+## Tag Release ##
+
+This will happen on Friday after any style tweaks land.
+
+1. https://github.com/mozilla-services/pageshot/releases/new
+2. Tag Version: YYYY-MM-DD (append -N if more than one release is tagged on a given day: 2016-04-08-1)
+3. Release Title: YYYY-MM-DD
```

Comment:
_set_package_version replaces the last digit in the version with the date, so let's use x.y.0 for the version numbers.